compiler/testData/codegen/box/notNullAssertions/messageLength.kt

```diff
@@ -0,0 +1,31 @@
+// TARGET_BACKEND: JVM
+// FILE: test.kt
+
+fun test(): String {
+    return A()
+        // Large comment that would cause the message for
+        // the non-null assertion to be very large if
+        // it is included verbatim in the message.
+        .foo()
+}
+
+fun box(): String {
+    try {
+        test()
+    } catch(e: Throwable) {
+        return if (e.message!!.length <= " must not be null".length + 50) {
```

Comment:
Just curious what the exception message is in this particular example.

Reply:
The exception message ends up being:

```
A()
        // Large com…e message.
        .foo() must not be null
```